README.md

```diff
@@ -13,13 +15,47 @@ For full info, docs and examples, see the
 [Meteor MessageFormat home page](http://messageformat.meteor.com/)
 (or install/clone the smart package and run `mrt` in its `website` directory).
 
-In short, you'll get to use the `{{mf}}` helper.  Simple example:
+Installation
+------------
+To install, simply run the following command in your project folder:
 
-```html
+```
+meteor add gadicc:messageformat
+```
+
+You will then need to add an initialization setting in your project. E.g. in `lib/config.js` add the following:
+
+```
+mfPkg.init('en');
+```
+Replace **'en'** with the native language of your project, if appropriate.
+
+Usage
+----
+Once installed, you can start adding the MessageFormat helper to templates:
+
+```
+<p>This string is translatable</p>
+```
+
+...becomes ...
+```
 <p>{{mf 'trans_string' 'This string is translatable'}}</p>
```

Comment:
It might be a good idea to mention that developer is responsible for uniqueness of 'trans_string' (the keys).